ruby/sinatra/Gemfile.lock

```diff
@@ -0,0 +1,41 @@
+GEM
```

Comment:
the lockfile is wrong... it installs wrong versions of our library (``ddtrace (0.1.5)``)